src/components/access_group/AccessGroupTerms.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <main class="group-terms">
+    <a
+      class="button group-terms__back-action"
+      :href="backUrl"
+      @click="handleBackClicked"
+      v-if="!invitationShowTOSAccept"
+    >
+      <Icon id="chevron-left" :width="17" :height="17" />{{
+        fluent('access-group_terms', 'back-action')
+      }}
+    </a>
+    <section class="primary-area">
+      <h1 class="group-terms__header">
+        {{ fluent('access-group_terms', 'heading') }}
+      </h1>
+      <p>{{ termsContent }}</p>
+      <footer class="group-terms__form" v-if="invitationShowTOSAccept">
+        <div class="form-action-row">
+          <input
+            type="radio"
+            class="form-action-row__input"
+            :value="true"
+            v-model="termsAccepted"
+          />
+          {{ fluent('access-group_terms', 'do-accept') }}
+        </div>
+        <div class="form-action-row">
+          <input
+            type="radio"
+            class="form-action-row__input"
+            :value="false"
+            v-model="termsAccepted"
+          />
+          {{ fluent('access-group_terms', 'do-not-accept') }}
+        </div>
+        <Button class="button--primary" @click="handleSubmitClicked">{{
+          fluent('access-group_terms', 'submit-changes')
+        }}</Button>
+      </footer>
+    </section>
+  </main>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex';
+import Icon from '@/components/ui/Icon.vue';
+import Button from '@/components/ui/Button.vue';
+
+export default {
+  name: 'AccessGroup',
+  components: {
+    Icon,
+    Button,
+  },
+  props: {
+    groupname: String,
+  },
+  methods: {
+    ...mapActions({
+      acceptInvitation: 'userV2/acceptInvitation',
+      rejectInvitation: 'userV2/rejectInvitation',
+    }),
+    handleBackClicked() {
+      console.log('Back clicked');
+    },
+    handleSubmitClicked() {
```

Comment:
Would this be cleaner as `handleTerms` with true/false?

Reply:
Perhaps. Personally, I think it's cleaner to have the function be directly related to the the action name. What are you doing? You're submitting the form. Personal preference :)